components/Hero/Hero.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Image from "next/image";
+import { Link } from "components/Link/Link";
+import { CTA, Photo } from "types/layout-schema-types";
+import { urlFor } from "lib/client";
+
+export interface HeroSection extends React.HTMLProps<HTMLElement> {
+  cta: CTA;
+  description: string;
+  photo: Photo;
+  title: string;
+}
+
+export const Hero = ({ title, description, photo, cta }: HeroSection) => {
+  return (
+    <section className="py-10">
+      <div className="flex max-w-screen-xl flex-col lg:mx-auto lg:flex-row lg:items-center">
+        <article className="mb-14 lg:mb-0 lg:w-5/12">
```

Comment:
@nykka what tag do you think is the best to represent this? 
Personally, I don't think that an `article` is the best one for this. I would use a `section` or just a `div` (after seeing some online examples, most of them they just use a div).

Reply:
Yes, that is not semantically accurate

Could you revert the commit 76d8e78 @alexxuss1996?

Reply:
Done @TonyMckes